http.js

```diff
@@ -23,36 +23,38 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  */
 
-const request = require('request');
+const axios = require('axios').default;
+const stringify = require("qs").stringify;
+
 
 exports.get = (getUrl, params, callback, redirects) => {
-    const options = {
-        url: getUrl,
+    axios.get(getUrl, {
         maxRedirects: redirects || 5,
         qs: params,
-        headers: { 'Accept' : 'application/xrds+xml,text/html,text/plain,*/*;q=0.9' }
-    };
-    request.get(options, function (error, response, body) {
-        if (error) {
-        callback(error);
-        } else {
-        callback(body, response.headers, response.statusCode);
-        }
+        headers: {
+            'Accept': 'application/xrds+xml,text/html,text/plain,*/*;q=0.9'
+        },
+        transformResponse: a => a
```

Comment:
Can be removed I think?

Reply:
I'm pretty sure that I added that because by default axios tries to parse the response as JSON, and that caused some errors for me